staging/nvidia/charts/grafana/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
```

Comment:
Would you mind adding details to the description as to what this subchart does 😄